src/Services/AuthorizationService.php

```diff
@@ -0,0 +1,43 @@
+<?php
+declare(strict_types=1);
+
+namespace LotGD\Crate\GraphQL\Services;
+
+use Symfony\Component\DependencyInjection\ContainerAwareTrait;
+
+use LotGD\Crate\GraphQL\AppBundle\GraphQL\Types\UserType;
+use LotGD\Crate\GraphQL\Models\User;
+
+class AuthorizationService extends BaseManagerService
+{
+    use ContainerAwareTrait;
+
+    protected function getCurrentUser(): ?User
+    {
+        $user = $this->container->get('security.token_storage')->getToken()->getUser();
+
+        if ($user instanceof User) {
+            return $user;
+        } else {
+            return null;
+        }
+    }
+
+    public function isUser($object) {
```

Comment:
Didn't I completely remove that part? Wording was temporary anyway, but you're right.